extensions/gdx-setup/src/com/badlogic/gdx/setup/data/extensions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+    <extension>
+       <name>Overlap2D</name>
+       <description>level and ui editor runtime</description>
+       <package>com.underwaterapps.overlap2druntime</package>
+       <version>0.0.8</version>
+       <dependencies>
+           <core>com.underwaterapps.overlap2druntime:overlap2d-runtime-libgdx</core>
+       </dependencies>
+    </extension>
+</extensions>
```

Comment:
I think we should keep them in just so it's a little clearer to other people wanting to add their extension.

Omitting the line completely if there are no additional dependencies and keeping the line if its not supported is a little weird.  I'd rather keep it verbose, and perhaps even commented, to avoid confusion.